tests/test_other.py

```diff
@@ -130,14 +130,33 @@ def metafunc(self, wasmfs):
   return metafunc
 
 
+WASMFS_BACKEND_MAP = {'': 'WASMFS_MEMORY_BACKEND',
+                      'node': 'WASMFS_NODE_BACKEND'}
```

Comment:
I feel like maybe exacting this map makes the two `metafunc._parameterize` below hard to read.. maybe just duplicate it instead?    Feel free to disagree ..

Reply:
Yeah, maybe better to be DAMP than DRY here.